index.js

```diff
@@ -1,14 +1,18 @@
 const StyleLintPlugin = require('stylelint-webpack-plugin');
 
-function createRewireStylelint(options = {}) {
-  return function(config, env) {
+const createRewireStylelint = function (opts = {}) {
+  return function (config, env, options) {
```

Comment:
Why would you want use options here ? options should be use only on create rewire function...

Reply:
For example, the following code contains a use case to justify this approach:

```js

Reply:
ok but, if you want use configs, you must use this way:

```
module.exports = function override(config, env) {
  config = rewireStylelint.withLoaderOptions({
    files: 'src/**/*.css',
  )(config, env);

  return config;
};

```

Reply:
Ah, ok! It seems that the documentation was broken in yesterday version. Thanks!